cqf-fhir-utility/src/main/java/org/opencds/cqf/fhir/utility/client/ExpandRunner.java

```diff
@@ -96,8 +96,10 @@ public void run() {
 
                 var expandedValueSetAdapter = (IValueSetAdapter) createAdapterForResource(expandedValueSet);
 
-                if (expandedValueSetAdapter.getExpansionTotal()
-                        > terminologyServerClientSettings.getExpansionsPerPage()) {
+                if ((expandedValueSetAdapter.getExpansionTotal()
+                                > terminologyServerClientSettings.getExpansionsPerPage())
```

Comment:
Why would we need to check the setting here? Either the expansion we're looking at is paged and we need to request the next page, or it's not, the setting is irrelevant, right?